service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/DocumentReferenceToNarrativeStatementMapper.java

```diff
@@ -63,6 +63,16 @@ public String mapDocumentReferenceToNarrativeStatement(final DocumentReference d
         return TemplateUtils.fillTemplate(NARRATIVE_STATEMENT_TEMPLATE, builder.build());
     }
 
+    public String buildBindingDocumentNarrativeStatement(String bindingDocumentId) {
+        final NarrativeStatementTemplateParametersBuilder builder = NarrativeStatementTemplateParameters.builder()
+            .narrativeStatementId(bindingDocumentId)
+            //.availabilityTime(getAvailabilityTime(documentReference))
```

Comment:
Why commented out? If this will be done in the future then add a TODO pointing to JIRA ticket